lib/autocheck/include/autocheck/generator.hpp

```diff
@@ -2,6 +2,7 @@
 #define AUTOCHECK_GENERATOR_HPP
 
 #include <random>
+#include <stdrandom.h>
```

Comment:
you need to do `#include "lib/util/stdrandom.h` like in other places (does not build on Windows otherwise)

Reply:
fixed.